easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.30-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,114 @@
+easyblock = 'Bundle'
+
+name = 'FusionCatcher'
+local_commit = '4e717c4a78f3ecf4c2ee6a59f9175b98d5040caf'
+version = '1.30'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ndaniel/fusioncatcher'
+description = """FusionCatcher searches for novel/known somatic fusion genes, translocations,
+and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS
+platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples."""
+github_account = 'ndaniel'
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAT', '3.5'),  # preferred over Kent_tools as BLAT is built from source
+    ('Bowtie', '1.2.3'),
+    ('Bowtie2', '2.3.5.1'),
+    ('BWA', '0.7.17'),
+    ('Kent_tools', '401'),  # required for liftOver
+    ('SRA-Toolkit', '2.10.4'),
+    ('STAR', '2.7.2b'),
+    ('picard', '2.21.6', '-Java-11', True),  # optional
+    ('Velvet', '1.2.10', '-mt-kmer_191'),  # optional
+    ('openpyxl', '2.6.4', versionsuffix),  # optional
+    ('parallel', '20190922'),  # optional
+    ('pigz', '2.4'),
+    ('zlib', '1.2.11'),
+]
+
+local_bbmap_makefile = 'makefile.%s' % {'Darwin': 'osx', 'Linux': 'linux'}[OS_TYPE]
+
+default_component_specs = {
+    'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+    'start_dir': '%(namelower)s-%(version)s',
+}
+
+components = [
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': [GITHUB_SOURCE],
+        'checksums': ['a6313ebee6f7225fa873d901e4c9ff7e0990763cde754df8db40d10523a69800'],
```

Comment:
@boegel checksum needs to be updated?

Reply:
Ah, yes, the name of the directory in the source tarball changed when switching to the properly tagged release...